common/src/gaussian.cpp

```diff
@@ -157,19 +157,19 @@ pcl::GaussianKernel::convolveRows (const pcl::PointCloud<float>& input,
   }
   
   std::size_t i;
-  for (std::size_t j = 0; j < unaliased_input->height; j++)
+  for (std::size_t j = 0; j < static_cast<uindex_t>(unaliased_input->height); j++)
```

Comment:
Why not convert `i` and `j` to `index_t` or `uindex_t` instead?

Reply:
This was because `i` was involved in operations with `radius` which would also cause the signed-unsigned warning.

Reply:
Then choose one which causes least casts 😄